.github/workflows/csharp.yml

```diff
@@ -52,35 +56,40 @@ jobs:
       EVENT_TYPE: ${{ github.event_name }}
       CURRENT_BRANCH: ${{ github.head_ref || github.ref_name }}
     steps:
-      - uses: actions/checkout@v3
-      - name: Setup .NET
-        uses: actions/setup-dotnet@v1
-        with:
-          dotnet-version: 5.0.x
-      - name: Restore nuget packages
-        run: |
-          nuget restore OptimizelySDK.Travis.sln
-          nuget install ./OptimizelySDK.Tests/packages.config -OutputDirectory ./packages
-          nuget install NUnit.Runners -Version 2.6.4 -OutputDirectory ./testrunner
-      - name: script
-        run: |
-          ./install_mono.sh
-          xbuild /p:SignAssembly=true /p:AssemblyOriginatorKeyFile=$(pwd)/keypair.snk /p:Configuration=Release ./OptimizelySDK.Travis.sln
-          mono ./testrunner/NUnit.Runners.2.6.4/tools/nunit-console.exe ./OptimizelySDK.Tests/bin/Release/OptimizelySDK.Tests.dll
-      - name: find and sign dll
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1
+      - name: Setup NuGet
+        uses: NuGet/setup-nuget@v1
+      - name: Restore NuGet packages
+        run: nuget restore ./OptimizelySDK.Travis.sln
+      - name: Build solution
+        run: msbuild /p:SignAssembly=true /p:AssemblyOriginatorKeyFile=$(pwd)/keypair.snk /p:Configuration=Release ./OptimizelySDK.Travis.sln
+      - name: Install NUnit Console
+        run: nuget install NUnit.Console -Version 3.15.2 -DirectDownload -OutputDirectory .
+      - name: Run NUnit tests
+        run: ./NUnit.ConsoleRunner.3.15.2\tools\nunit3-console.exe /timeout 10000 /process Separate ./OptimizelySDK.Tests/bin/Release/OptimizelySDK.Tests.dll
```

Comment:
Good to have, if you can specify each parameter what it does. 
e.g. /process Separate and /timeout, or add a link to see what parameters are supported.

Reply:
Oh, that's a great idea. Yes. I'll add a comment.